views/libraries/selectLocation.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<!--
+ @license
+ Copyright 2019 Google LLC. All Rights Reserved.
+ SPDX-License-Identifier: Apache-2.0
+-->
+<html>
```

Comment:
I get a warning typically from html validator when I don't include lang="en". Maybe add it just to be safe.